cmiles/utils.py

```diff
@@ -368,6 +421,21 @@ def has_explicit_hydrogen(molecule):
     return toolkit.has_explicit_hydrogen(molecule)
 
 
+def add_explicit_hydrogen(molecule):
+    """
+    Add explicit hydrogen to molecule
+    Parameters
```

Comment:
Blank line before Parameters and if Returns are not set (like below), you should remove that area to avoid potential errors when rendering the docs.